### DISCUSSION TITLE
Pre-push linting hook to strict ?

### DISCUSSION THREAD

**Discussion by game5413** [nanoexpress/middlewares#33] [Q&A] [answered]
Hi @dalisoft , i wonder that Pre-push hook is to strict or what because trying push into new branch, got rejected because prettier, after tried pretty that file mentioned still got same result.
![image](https://user-images.githubusercontent.com/37659721/130312922-067e213b-817e-422b-8d0f-9bc72e436f20.png)

**Comment by dalisoft** [comment, marked as answer]
@game5413 Seems has really format error in these files, please fix them first or this maybe **prettier** bug. Please check with formatting these files via `prettier ./packages --write` command in terminal

**Reply by game5413** [reply]
Nothing file changed, but it solve the problem. Maybe that was **prettier** bug.

**Reply by dalisoft** [reply]
Nice, if it was fixed itself
